requirements/base.in

```diff
@@ -18,3 +18,15 @@
 -e file:.
 pyrsistent>=0.16.1,<0.17
 zipp==3.4.1
+
+# stubs
```

Comment:
Do we need these after the update to pre-commit?

Reply:
Good point, probably not